src/constants/projectIds.ts

```diff
@@ -0,0 +1,3 @@
+export const CONSTITUTIONDAO_PROJECT_ID = 36
```

Comment:
thanks for cleaning up this system 😁 just a nit pick, but wdyt about making this an object like this?

```
PROJECT_IDS = {
  CONSTITUTION_DAO: 36
}
```

Reply:
Love it, will do